lib/zen_desk/client.rb

```diff
@@ -7,15 +7,19 @@ def initialize(ticket_class: ZendeskAPI::Ticket, client: nil)
       @client = client
     end
 
-    def create_ticket(name:, email:, message:, subject:, tags:)
-      ticket_class.create!(
-        client,
+    # rubocop:disable Metrics/ParameterLists
+    def create_ticket(name:, email:, message:, subject:, tags:, custom_fields: nil)
+      packet = {
```

Comment:
Not sure `packet` is such a good name for describing this thing. Maybe `zendesk_message` or `zendesk_ticket` or even just `ticket`?